docs/pages/2020_Intro_Week/sections/workflows_adv.rst

```diff
@@ -73,35 +85,35 @@ If the workflow does not explicitly check for this case, but instead blindly ass
 
 .. figure:: include/images/workflow_error_handling_basic_failed.png
 
-    Example execution of the Quantum ESPRESSO phonon workflow where the third step, the ``q2r.x`` code, failed and because the workflow blindly assumed it would have finished without errors also fails.
+    Example execution of the Quantum ESPRESSO phonon workflow where the third step, the ``q2r.x`` code, failed, and because the workflow blindly assumed it would have finished without errors also fails.
 
 The solution seems simple then.
 After each calculation, we simply add a check to verify that it finished successfully and produced the required outputs before continuing with the next calculation.
 What do we do, though, when the calculation failed?
 Depending on the cause of the failure, we might actually be able to fix the problem, and re-run the calculation, potentially with corrected inputs.
-A common example is that the calculation ran out of walltime and was cancelled by the job scheduler.
+A common example is that the calculation ran out of walltime (scheduled run time on a remote cluster) and was cancelled by the job scheduler.
```

Comment:
Yeah, I thought this might come back to bite me in the ...
How would you describe walltime then? Just the user-specified dedicated CPU run time? Or something?

Reply:
Maybe "requested run time from the job scheduler" ?